### PR TITLE
CORE-6684: Custom Serializer Loading for Simulator

### PR DESCRIPTION
**Pull request by ashutoshmeher-r3**

### CODE REVIEW

**Review by mattbradburyr3 — DISMISSED** [review]
Looks good to me, but I'd like Liz to okay it.

File: simulator/runtime/src/test/kotlin/net/corda/simulator/runtime/config/DefaultConfigurationBuilderTest.kt [hunk 1]
```diff
@@ -42,6 +43,19 @@ class DefaultConfigurationBuilderTest {
         }
     }
 
+    @Test
+    fun `should allow registration of custom serializer via configuration`(){
```
Comment (fenryka) [inline simulator/runtime/src/test/kotlin/net/corda/simulator/runtime/config/DefaultConfigurationBuilderTest.kt]: this doesn't test what the test name says

**Review by fenryka — CHANGES_REQUESTED** [review]
One comment about test names

File: simulator/runtime/src/test/kotlin/net/corda/simulator/runtime/config/DefaultConfigurationBuilderTest.kt (hunk 1 above)
Comment (ashutoshmeher-r3) [reply simulator/runtime/src/test/kotlin/net/corda/simulator/runtime/config/DefaultConfigurationBuilderTest.kt]: updated the test name to "should be able to add custom serializers to configuration"

**Review by LizR3 — APPROVED** [review]

File: simulator/runtime/src/test/kotlin/net/corda/simulator/runtime/config/DefaultConfigurationBuilderTest.kt (hunk 1 above)
Comment (ashutoshmeher-r3) [reply simulator/runtime/src/test/kotlin/net/corda/simulator/runtime/config/DefaultConfigurationBuilderTest.kt]: @fenryka Could you approve this please?

**Pull request closed without merging**